parser.go

```diff
@@ -833,16 +833,25 @@ func expandWithExpr(was []*withArgExpr, e Expr) (Expr, error) {
 			// Already expanded.
 			return t, nil
 		}
+		metricName := ""
 		{
 			var me MetricExpr
 			// Populate me.LabelFilterss
+			// Check to see if name is set in the first labelFilter
```

Comment:
```suggestion
			// Check to see if name is set in the first labelFilter
```

Move this stray comment to relevant location